crates/packager/src/config/mod.rs

```diff
@@ -631,8 +631,8 @@ impl Default for LogLevel {
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct Binary {
-    /// File name and without `.exe` on Windows
-    pub filename: String,
+    /// Path to the binary. If it's relative, it will be resolved from [`Config::out_dir`].
+    pub path: PathBuf,
```

Comment:
How does this handle `.exe` on Windows? the reason why I didn't use path in the first place is because I wanted users to specify the `filename` and then we add `.exe` on Windows if needed, using a path, breaks this goal.

Reply:
it's still the same thing, you still just skip the extension, it doesn't change that behavior. but now you can specify absolute paths which is needed for electron

Reply:
I see, it is my fault, I missed a couple of places where I needed to call `.with_extension("exe")`, pushed a fix